internal/commands/output.go

```diff
@@ -377,3 +384,67 @@ func (t *TableOutputManager) Flush() error {
 
 	return nil
 }
+
+// JUnitOutputManager formats its output as a JUnit test result
+type JUnitOutputManager struct {
+	p      parser.Package
+	writer io.Writer
+}
+
+// NewDefaultJUnitOutputManager creates a new JUnitOutputManager using standard out
+func NewDefaultJUnitOutputManager() *JUnitOutputManager {
+	return NewJUnitOutputManager(os.Stdout)
+}
+
+// NewJUnitOutputManager creates a new JUnitOutputManager with a given Writer
+func NewJUnitOutputManager(w io.Writer) *JUnitOutputManager {
```

Comment:
Adding some test cases would be nice. A basic test covering the different results (pass, fail, warn) in `output_test.go` would be great.

Reply:
I'm must have been sleepy, not sure how I missed this file. I added some tests :)